app/feed/page.tsx

```diff
@@ -15,6 +19,24 @@ type Props = {
 };
 
 export default async function FeedPage({ searchParams }: Props) {
+  const filterEvetns = [
```

Comment:
Ensure proper spellings

```suggestion
  const filterEvents = [
```